### DISCUSSION TITLE
Need developer to build Milvus database

### DISCUSSION THREAD

**Discussion by jaylmilan** [milvus-io/milvus#16113] [Q&A and General discussion]
I am looking for experienced Milvus developer to build database for similarity search. Data size is 50GB to start, need to scale to 550GB. Please PM me if interested. Thanks

**Comment by xiaofan-luan** [comment]
Good luck @jaylmilan , if you hit any trouble on using milvus let us know

**Reply by jaylmilan** [reply]
Thanks @xiaofan-luan for your kind words. I am having trouble with milvus memory. My raw data size is 7GB, number of vectors in my raw data is 3.22M, 440 dimensions, nlist value 65536 and index type is IVF_FLAT. Total Milvus memory should be 23GB (see attached picture), but it's taking 136GB of my disk space. I couldn't scale my indexing due to this abnormal memory usage. Would love some suggestion on reducing milvus memory usage to normal levels. Thanks


![Picture2](https://user-images.githubusercontent.com/84993840/159178280-ea0386a7-5900-4d02-b8a4-1d6ec0306913.png)
